h/static/scripts/directive/annotation.coffee

```diff
@@ -371,6 +379,8 @@ AnnotationController = [
         else
           drafts.add model, => this.revert()
 
+      setPermissions()
```

Comment:
setPermissions() is called just prior to `render()` which recreates the view model (`@annotation`) from the domain model - so I don't think it is necessary to modify `@annotation` in `setPermissions()`